lib/logging/logging.dart

```diff
@@ -168,6 +180,20 @@ class LoggingScreen extends Screen {
         e.timestamp,
         summaryHtml: '$frameInfo$div',
       ));
+    } else if (e.extensionKind == 'Flutter.Error') {
+      final RemoteDiagnosticsNode node =
+          RemoteDiagnosticsNode(e.extensionData.data, objectGroup, false, null);
+      if (_verboseDebugging) {
```

Comment:
remove XXX from this code.
I use XXX to indicate code that should be reviewed before checking in.

Reply:
👍

Reply:
Done.